app/about/page.tsx

```diff
@@ -6,86 +6,146 @@ const DEFAULT_IMAGE = "/icons/app/stellarHorizons.png" as const;
+            <div className="mb-4 ">
+              <label htmlFor="email" className="block text-sm font-medium text-gray-700 mb-1">
+                Email
+              </label>
+              <input
+                type="email"
+                id="email"
+                className="w-full border border-gray-300 rounded-md p-2 focus:ring-2 focus:ring-blue-500"
+              />
+            </div>
+            <div className="mb-4 ">
+              <label htmlFor="message" className="block text-sm font-medium text-gray-700 mb-1">
+                Message
+              </label>
+              <textarea
+                id="message"
+                rows={4}
+                className="w-full border border-gray-300 rounded-md p-2 focus:ring-2 focus:ring-blue-500"
+              ></textarea>
+            </div >
+            <div className="flex justify-center">
+              <button
+                type="submit"
+                className=" bg-black text-white py-2 px-20 rounded-md hover:bg-blue-600 w-auto"
+              >
+                Submit
+              </button>
+            </div>
+          </form>
+        </div>
 
-      {/* Team Cards */}
-      <section className="flex justify-center">
-        <div className="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4 gap-8 place-items-center">
-          {team.map((member, index) => (
-            <div
-              key={index}
-              className="bg-white p-6 rounded-lg shadow-lg text-center"
-            >
+        {/* Meet Our Team */}
+        <div className="flex-[1] lg:flex-[0.6]">
+          <section className="text-center mb-6">
+            <h2 className="text-3xl font-bold mb-4">Meet Our Team</h2>
+            <p className="text-lg">
+              We are a group of passionate developers committed to building the best hotel-booking platform.
+            </p>
+          </section>
+
+          <section className="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-2 gap-6">
+            {team.map((member, index) => (
+              <div
+                key={index}
+                className="bg-white p-4 rounded-lg shadow-lg text-center"
+              >
               <Image
                 src={member.image}
-                alt={member.name}
-                className="w-32 h-32 rounded-full mx-auto mb-4"
+                alt={`${member.name}'s picture`}
+                className="w-24 h-24 rounded-full mx-auto mb-4"
                 width={100}
                 height={100}
+                quality={100}
               />
-              <h3 className="text-2xl font-semibold mb-2">{member.name}</h3>
-              <p>{member.description}</p>
-            </div>
-          ))}
+                <h3 className="text-xl font-semibold mb-2">{member.name}</h3>
+                <p>{member.description}</p>
+                <p className="text-sm text-gray-600">
+                  <a href={`mailto:${member.email}`} className="hover:text-blue-500">
```

Comment:
I think I linkedin link would be make more sense. I wouldn't want to give out my email.